# HG changeset patch
# User Alexander Dreyer <adreyer@gmx.de>
# Date 1344342695 -7200
# Node ID 59c467b8a840ae2f4975bdda847e71caebf54845
# Parent  521e344431d4e613cdcf5d6dda2ec8ead64b547a
Fix: doc/python depend on python module

diff -r 521e344431d4e613cdcf5d6dda2ec8ead64b547a -r 59c467b8a840ae2f4975bdda847e71caebf54845 SConstruct
--- a/SConstruct	Mon Aug 06 16:08:13 2012 +0200
+++ b/SConstruct	Tue Aug 07 14:31:35 2012 +0200
@@ -1279,6 +1279,7 @@
 
         env.Clean(pydocu, glob(PyRootPath('polybori/*.pyc')) +
                   glob(PyRootPath('polybori/dynamic/*.pyc')))
+        env.Depends(pydocu, PyRootPath('polybori/dynamic/__init__.py'))
 #        env.Ignore(pydocu, dynamic_modules)
     #bld=Builder("cd")

